aten/src/ATen/templates/TensorBody.h

```diff
@@ -976,6 +976,57 @@ struct MaybeOwnedTraits<at::Tensor> {
     return true;
   }
 };
+
+template <>
+struct ExclusivelyOwnedTraits<at::Tensor> {
+  using repr_type = at::Tensor;
+  using pointer_type = const at::Tensor*;
+
+  static repr_type nullRepr() {
+    // REVIEW: do we need to be able to distinguish an empty
+    // ExclusivelyOwned<Tensor> from an undefined Tensor?
```

Comment:
seems unlikely

Reply:
Alternately, we could delete the default constructor from ExclusivelyOwned (I am not sure how painful that will be for consumers)